gesture.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>Gesture Liveness Detection</title>
+</head>
+<body>
+  <h1>Gesture Liveness Detection</h1>
+  <h2 id="gesturePrompt"></h2> 
+  <video id="video" autoplay playsinline width="640" height="480" style="border:1px solid black;"></video>
+
+  <script type="module">
+    import Human from 'https://cdn.jsdelivr.net/npm/@vladmandic/human@latest/dist/human.esm.js';
+
+    async function run() {
+      const allGestures = ['thumbs up', 'open palm', 'facing left', 'facing right', 'head up', 'blink left eye', 'blink right eye', 'index up', 'pinky up'];
```

Comment:
You may want to consider making this an array of objects, so you can have a prompt label, and the Human label linked together. This would give you more flexibility in providing a prompt to the user - e.g.
```
{
    gesture: 'open palm',
    prompt: 'Show your open palm'
}